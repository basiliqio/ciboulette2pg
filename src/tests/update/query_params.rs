use super::*;

ciboulette_query_test_single!(test_update);
ciboulette_query_test_relationship_many_to_one!(test_update_many_to_one);
