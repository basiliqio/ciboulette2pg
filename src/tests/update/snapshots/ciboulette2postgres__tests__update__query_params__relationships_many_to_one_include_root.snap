---
source: src/tests/update/query_params.rs
expression: value

---
[
  {
    "id": "[uuid]",
    "type": "comments",
    "data": {
      "body": "Wasnt convinced..."
    }
  },
  {
    "id": "[uuid]",
    "type": "peoples",
    "related_type": "comments",
    "related_id": "[uuid]"
  }
]
